awx/ui/src/components/Schedule/shared/ScheduleFormFields.js

```diff
@@ -0,0 +1,194 @@
+import DateTimePicker from './DateTimePicker';
+import sortFrequencies from './sortFrequencies';
+
+const SelectClearOption = styled(SelectOption)`
+  & > input[type='checkbox'] {
+    display: none;
+  }
+`;
+
+export default function ScheduleFormFields({
+  hasDaysToKeepField,
+  zoneOptions,
+  zoneLinks,
+}) {
+  const [timezone, timezoneMeta] = useField({
+    name: 'timezone',
+    validate: required(t`Select a value for this field`),
+  });
+  const [frequency, frequencyMeta, frequencyHelper] = useField({
+    name: 'frequency',
+    validate: required(t`Select a value for this field`),
+  });
+  const [timezoneMessage, setTimezoneMessage] = useState('');
+  const warnLinkedTZ = (event, selectedValue) => {
+    if (zoneLinks[selectedValue]) {
+      setTimezoneMessage(
+        t`Warning: ${selectedValue} is a link to ${zoneLinks[selectedValue]} and will be saved as that.`
+      );
+    } else {
+      setTimezoneMessage('');
+    }
+    timezone.onChange(event, selectedValue);
+  };
+  let timezoneValidatedStatus = 'default';
+  if (timezoneMeta.touched && timezoneMeta.error) {
+    timezoneValidatedStatus = 'error';
+  } else if (timezoneMessage) {
+    timezoneValidatedStatus = 'warning';
+  }
+  const config = useConfig();
+
+  // const [exceptionFrequency, exceptionFrequencyMeta, exceptionFrequencyHelper] =
+  //   useField({
+  //     name: 'exceptionFrequency',
+  //     validate: required(t`Select a value for this field`),
+  //   });
+
+  const updateFrequency = (setFrequency) => (values) => {
+    setFrequency(values.sort(sortFrequencies));
+  };
+
+  return (
+    <>
+      <FormField
+        id="schedule-name"
+        label={t`Name`}
+        name="name"
+        type="text"
+        validate={required(null)}
+        isRequired
+      />
+      <FormField
+        id="schedule-description"
+        label={t`Description`}
+        name="description"
+        type="text"
+      />
+      <DateTimePicker
+        dateFieldName="startDate"
+        timeFieldName="startTime"
+        label={t`Start date/time`}
+      />
+      <FormGroup
+        name="timezone"
+        fieldId="schedule-timezone"
+        helperTextInvalid={timezoneMeta.error || timezoneMessage}
+        isRequired
+        validated={timezoneValidatedStatus}
+        label={t`Local time zone`}
+        helperText={timezoneMessage}
+        labelIcon={<Popover content={helpText.localTimeZone(config)} />}
+      >
+        <AnsibleSelect
+          id="schedule-timezone"
+          data={zoneOptions}
+          {...timezone}
+          onChange={warnLinkedTZ}
+        />
+      </FormGroup>
+      <FormGroup
+        name="frequency"
+        fieldId="schedule-frequency"
+        helperTextInvalid={frequencyMeta.error}
+        validated={
+          !frequencyMeta.touched || !frequencyMeta.error ? 'default' : 'error'
+        }
+        label={t`Repeat frequency`}
+      >
+        <FrequencySelect
```

Comment:
When selecting multiple frequencies the `Repeat Frequency` detail in the Schedule Detail page may be a bit misleading:
<img width="573" alt="Screen Shot 2022-07-20 at 3 18 19 PM" src="https://user-images.githubusercontent.com/2293210/180092584-6cdc4c2f-d506-4096-9ca8-28f09d40b4de.png">

I have a schedule that repeats every min and hour, however the `Repeat Frequency` detail reads "every year."